### ISSUE TITLE
CLI: coffee --watch fails on file that does not exist

### ISSUE THREAD

**Issue by ianb** [jashkenas/coffeescript#985]
When using `coffee --watch` I have non-existent files in the directory (Emacs creates files like `.#filename.coffee`).  It would be better to simply ignore these.  Sample traceback:

```
/home/ianb/.node_libraries/.npm/coffee-script/1.0.0/package/lib/command.js:62
        throw new Error("File not found: " + source);
              ^
Error: File not found: static/.#site.coffee
  at /home/ianb/.node_libraries/.npm/coffee-script/1.0.0/package/lib/command.js:62:19
  at path.js:151:19
```

**Comment by VbifRkbvjd** [comment]
This issue appears in coffeescript 1.8.0 too.
File not found:   .#screenshot.coffee
And watch then stops. 
I am using command :
```
coffee --watch -o output -c ./ 
```  
Strange thing, that this is also takes into account non coffee files. like
```
File not found: .#postinstall.sh
```

**Comment by philc** [comment]
Can we reopen this bug? It has been regressed recently. I've reproduced this in coffeescript 1.9.3 and 1.10, and @MichaelLeachim saw it in 1.8.0.

It's a very annoying bug for emacs developers, because operations within emacs which create temporary files and then delete them (like using magit to write a git commit message) crashes the coffeescript process.

**Comment by celwell** [comment]
This has been annoying me as well. Is there a filter I can add to my watch dir command?

`coffee -o . -cwb src`

**Comment by GeoffreyBooth** [comment]
I can’t seem to reproduce this. I tried running `emacs` and creating and saving, and opening and saving files. I also ran `coffee -cwo ./dist/ ./src` in one tab and `touch .foo.coffee` in another tab, and then `rm .foo.coffee` in that tab, and the watch task handled the events just fine:

```
19:33:50 - compiled ~/src/.foo.coffee
19:34:01 - removed ~/src/.foo.coffee
```

If someone can post steps to reproduce in current CoffeeScript, I’ll reopen.
